README.md

```diff
@@ -11,7 +11,7 @@ It is distributed via
 
      Lehrstuhl B fuer Mathematik
      RWTH-Aachen
-     Prof. Plesken
+     Prof. i.R. Plesken
```

Comment:
Shouldn't be capital `I` ?

Reply:
No @alex-konovalov , i.R. stands for the German "in Rente", which means retired.

Reply:
See [here](https://www.mathb.rwth-aachen.de/cms/MATHB/Der-Lehrstuhl/Team/Professorinnen-und-Professoren/~rnmq/Univ-Prof-i-R-Dr-rer-nat-Wilhelm-Pl/lidx/1/)